### PR TITLE
Localize all the flows

### PR DESCRIPTION
**Pull request by ericnewcomer**
* Remove old non-localized flow format, now all non-localized flows use 'base' as the default. 
* Move flow migration code to its own module

### CODE REVIEW

File: temba/flows/flow_migrations.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+from uuid import uuid4
+from temba.flows.models import ContainsTest, StartsWithTest, ContainsAnyTest, RegexTest, ReplyAction
+from temba.flows.models import SayAction, SendAction, RuleSet
+
+
+def migrate_to_version_6(json_flow):
+    """
+    This migration removes the non-localized flow format. This means all potentially localizable
+    text will be a dict from the outset. If no language is set, we will use 'base' as thew
+    default language.
```
Comment (nicpottier) [inline temba/flows/flow_migrations.py]: typo

**Comment by nicpottier** [comment]
So much better.

https://www.youtube.com/watch?v=7PCkvCPvDXk

**Comment by ericnewcomer** [comment]
1 billion views?!

**Comment by ericnewcomer** [comment]
Shall we merge? Speak now or forever hold your peas.

**Comment by nicpottier** [comment]
doit

**Pull request merged**